cattle/agent/handler.py

```diff
@@ -89,7 +89,7 @@ def _get_response_data(self, obj):
                 pass
 
             if method is not None:
-                return method(obj)
+                return {type: method(obj)}
```

Comment:
I just realized that I may be wrong that that the type for a container ='container'.  I need to double check that.  Regardless, you can't assume the object matches.  Please grab it from the resourceType.  This is guaranteed to match the logic in the AgentBasedProcessHandler.